gulpfile.js

```diff
@@ -27,7 +27,8 @@ gulp.task('mocha', function () {
   return gulp
     .src(['./test/**/*', '!./test/temp'])
     .pipe(mocha({
-      reporter: 'spec'
+      reporter: 'spec',
+      timeout: Infinity
```

Comment:
According to 'mocha' docs, 'timeout(0)' works only writhin suite. In global settings it sets timeout to 0 milliseconds - so I used infinity. That being said, I like 'timeout(0)' in each suite better: makes every suite self-sufficient.